.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,4 +10,16 @@ And the contributing guide: https://github.com/containous/traefik/blob/master/.g
 
 <!--
 Briefly describe the pull request in a few paragraphs.
--->
\ No newline at end of file
+-->
+
+<!--
+ PLEASE, DON'T REMOVE THIS CHECKLIST AND COMPLETE IT.
```

Comment:
Who is going to use this check list?

If it's going to be the maintainers, I'm generally fine with the structure.

If it's going to be the PR author, then I think a list without check boxes will be good enough. Otherwise, authors will just forget to tick them off, and we'll have the extra burden to remind/ask them or tick things off ourselves, which will just be forgotten as well.